package.json

```diff
@@ -15,50 +15,39 @@
     "test": "ember test"
   },
   "devDependencies": {
-    "autoprefixer": "6.3.7",
-    "broccoli-asset-rev": "2.4.2",
-    "ember-ajax": "2.0.1",
-    "ember-cli": "2.11.1",
-    "ember-cli-app-version": "1.0.0",
-    "ember-cli-babel": "6.4.1",
-    "ember-cli-bootstrap-sassy": "0.5.3",
-    "ember-cli-code-coverage": "0.3.11",
-    "ember-cli-dependency-checker": "2.0.1",
-    "ember-cli-eslint": "3.0.3",
-    "ember-cli-htmlbars": "2.0.2",
-    "ember-cli-htmlbars-inline-precompile": "0.4.3",
-    "ember-cli-inject-live-reload": "1.4.0",
-    "ember-cli-inline-content": "0.4.0",
-    "ember-cli-meta-tags": "3.0.1",
-    "ember-cli-moment-shim": "3.3.1",
-    "ember-cli-postcss": "3.0.5",
-    "ember-cli-qunit": "3.0.0",
-    "ember-cli-release": "0.2.9",
-    "ember-cli-sass": "5.3.1",
-    "ember-cli-shims": "1.0.2",
-    "ember-cli-sri": "2.1.0",
-    "ember-cli-template-lint": "0.4.12",
-    "ember-cli-test-loader": "1.1.0",
-    "ember-cli-uglify": "1.2.0",
-    "ember-data": "2.7.0",
-    "ember-export-application-global": "1.0.5",
-    "ember-font-awesome": "2.1.1",
-    "ember-get-config": "0.0.4",
-    "ember-load-initializers": "0.5.1",
     "@centerforopenscience/ember-osf": "0.7.1",
-    "ember-power-select": "1.0.0-beta.7",
-    "ember-resolver": "2.1.0",
-    "ember-sanitize": "2.0.2",
-    "ember-sortable": "1.8.2",
-    "eslint": "3.18.0",
-    "loader.js": "4.5.1",
-    "postcss": "5.1.0"
+    "broccoli-asset-rev": "^2.4.5",
```

Comment:
Do you prefer `^` and `~` over `x`, like `2.x.x`, `2.0.x`? I find the latter much easier to read.